db/migrate/20150309132532_add_luarocks.rb

```diff
@@ -0,0 +1,20 @@
+class AddLuaRocks < ActiveRecord::Migration
```

Comment:
Rails' autoloader trips over this.  Can you change the class name to AddLuarocks so that it matches the Rails naming conventions? You don't need to change anything in the class itself - just change that class name and Rails will be happy.